tests/test_functional.py

```diff
@@ -607,10 +607,10 @@ def test_obal_repoclosure():
 
     expected_log = [
         "dnf repoclosure",
+        "--newest",
+        "--refresh",
         "--config",
         "repoclosure/yum.conf",
-        "--newest",
-        "--check downloaded_rpms",
```

Comment:
it looks like now it did not call any `--check` repos, which is probably wrong?

Reply:
or what is that test testing? there is no `downloaded_rpms` fake created (like in L622)

Reply:
There is something weird going on with this test, as it does not appear to be testing for *anything*. I can change the expected log to "ahdslkjsdf" and it still passes. I am in investigating.